dcos_launch/templates/vpc-cluster-template.json

```diff
@@ -345,8 +349,13 @@
             "Key" : "Network",
             "Value" : "Public",
             "PropagateAtLaunch" : "true"
-            }
-          ]
+          },
+          {
```

Comment:
This tag is in the wrong Autoscale group in this template, use `BareServerAutoScale` instead of `BootstrapServerPlaceholderAutoScale` https://github.com/dcos/dcos-launch/blob/master/dcos_launch/templates/vpc-cluster-template.json#L284